src/components/Modals/Changelog/index.tsx

```diff
@@ -19,18 +17,39 @@ const fetchReleaseBody = async () => {
     });
 };
 
+const getReadmeHTML = async (readmeText: string) => {
```

Comment:
Maybe we should put this in one of the utils files if it's basically the same function as the readme modals?

Reply:
Makes sense yeah, gonna look into it.